system/include/libc/assert.h

```diff
@@ -16,6 +16,9 @@
 extern "C" {
 #endif
 
+#if __EMSCRIPTEN__
```

Comment:
I think this is the only place in libc, so just this one for now. Doesn't look like there are too many others in the project, but yeah, not in this PR.